index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles/index.css" />
+    <title>Microverse Portfolio</title>
+  </head>
+  <body>
+    <section class="nav-section relative">
+      <nav class="w-full">
+        <div class="flex align-items-center h-4">
+          <div class="nav-items text-center">
+            <a href="#"><span class="fw-600">Portfolio</span></a>
+            <a href="#"><span class="fw-600">About</span></a>
+            <a href="#"><span class="fw-600">Contact</span></a>
+          </div>
+          <div class="nav-container">
+            <input class="checkbox" type="checkbox" />
+            <div class="menu-button">
+              <span class=""></span>
+              <span class=""></span>
+              <span class=""></span>
+            </div>
+          </div>
+        </div>
+      </nav>
+    </section>
+
+    <div class="hero-bg absolute"></div>
+
+    <section
+      class="hero-section relative flex flex-col align-items-center h-screen"
+    >
+      <div class="hero-container">
+        <div class="hero-content fw-700 leading-4 w-full text-center">
+          <h3 class="text-dark hero-title">
+            Hey there. I'm Alex. <br />
+            <span class="text-primary">I'm a software developer</span>
+          </h3>
+        </div>
+        <p class="hero-desc mt-0 mb-2">
+          I can help you build a I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hesitate to contact
+          me.
+        </p>
+      </div>
+      <div class="social-link flex justify-center">
+        <a href=""
+          ><span> <img src="./assets/icons/twitter.svg" alt="twitter" /> </span
+        ></a>
+        <a href=""
+          ><span> <img src="./assets/icons/linkedin.svg" alt="twitter" /> </span
+        ></a>
+        <a href=""
+          ><span> <img src="./assets/icons/medium.svg" alt="twitter" /> </span
+        ></a>
+        <a href=""
+          ><span> <img src="./assets/icons/github.svg" alt="twitter" /> </span
+        ></a>
+        <a href="">
+          <span></span>
+          <img src="./assets/icons/angellist.svg" alt="angelist" />
+        </a>
+      </div>
```

Comment:
Nice work on laying out your social links with Flex 👍.  Let's make it semantic by using a `ul` and `li` tag to list our social handle👍  

```suggestion
      <ul class="social-link flex justify-center">
              <li>
                  <a href=""
                    ><span> <img src="./assets/icons/medium.svg" alt="twitter" /> </span
                  ></a>
                  <a href=""
                    ><span> <img src="./assets/icons/github.svg" alt="twitter" /> </span
                  ></a>
             </li>
              <li>
                  <a href=""
                    ><span> <img src="./assets/icons/medium.svg" alt="twitter" /> </span
                  ></a>
                  <a href=""
                    ><span> <img src="./assets/icons/github.svg" alt="twitter" /> </span
                  ></a>
             </li>
        
              <li>
                  <a href=""
                    ><span> <img src="./assets/icons/medium.svg" alt="twitter" /> </span
                  ></a>
                  <a href=""
                    ><span> <img src="./assets/icons/github.svg" alt="twitter" /> </span
                  ></a>
             </li>
           <li>
                <a href="">
                  <span></span>
                  <img src="./assets/icons/angellist.svg" alt="angelist" />
                </a>
            </li>
      </ul>
```

I guess the suggestion above makes it clearer. but dont forget to personalise the links👍